### PR TITLE
Report sbat settings in meson summary

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Review by yuwata — APPROVED** [review]
LGTM.

File: src/boot/efi/meson.build [hunk 1]
```diff
@@ -288,6 +291,16 @@ summary({
...
         section : 'Extensible Firmware Interface')
 
+if efi_conf.get('SBAT_DISTRO', '') != ''
+        summary({
+                'SBAT distro': efi_conf.get('SBAT_DISTRO'),
+                'SBAT distro generation': efi_conf.get('SBAT_DISTRO_GENERATION'),
+                'SBAT distro version': sbat_distro_version_display,
+                'SBAT distro summary': efi_conf.get('SBAT_DISTRO_SUMMARY'),
+                'SBAT distro URL': efi_conf.get('SBAT_DISTRO_URL')},
+                section : 'Extensible Firmware Interface')
```
Comment (medhefgo) [inline src/boot/efi/meson.build:300]: Might want to align these like above?

**Review by medhefgo — APPROVED** [review]
Looks good. If you add a sbat section to summary, the `message()` calls in sbat detection could be dropped imho.

**Comment by keszybz** [comment]
Updated with the suggested changes.

**Pull request merged**